python/setup.py

```diff
@@ -28,7 +28,7 @@
 else:
     from setuptools import setup
     from setuptools.extension import Extension
-    kwargs = {'install_requires': ['numpy<=1.15.0,>=1.8.2', 'requests<2.19.0,>=2.18.4', 'graphviz<0.9.0,>=0.8.1'], 'zip_safe': False}
+    kwargs = {'install_requires': ['numpy<=1.15.0,>=1.8.2', 'requests<2.19.0,>=2.18.4', 'graphviz<0.9.0,>=0.8.1', 'psutil>=5.4.3'], 'zip_safe': False}
```

Comment:
Thanks for fixing this.  I think Lam made a different fix as part of commit 1101c160d7d0612b59becb44e32b895adc877bca (docker/scripts/run-mx-ngraph-validation-test.sh).  Should this PR undo Lam's change?